bucket/guiscrcpy.json

```
@@ -0,0 +1,37 @@
+{
+    "##": "User config is stored at '$Env:LocalAppData\\guiscrcpy'",
+    "version": "2022.7.1",
+    "description": "A simple, pluggable, graphical user interface for the fastest Android screen mirroring software, scrcpy",
+    "homepage": "https://github.com/srevinsaju/guiscrcpy",
+    "license": "GPL-3.0-or-later",
+    "depends": "scrcpy",
+    "notes": "If you are required to specify scrcpy-server's location, you can find it in your scrcpy installation directory.",
```

```suggestion
    "notes": "If you are required to specify scrcpy-server's location, you can find it in $dir.",
```
Expands to the full path of the installation directory - more helpful for users.

---

$dir will return guiscrcpy’s installation path but this is about scrcpy’s

---

Sadly we can't expand Scoop's installation path in notes. But we could instead put: 
```suggestion
    "notes": "If you are required to specify scrcpy-server's location, you can find it in ~\scoop\apps\scrcpy\current\",
```
Those who installed Scoop globally will understand where to find it from this abbreviated path.